mokapot/confidence.py

```diff
@@ -106,15 +101,14 @@ def __init__(
             .drop_duplicates(psms.spectrum_columns, keep=keep)
             .index
         )
-
-        self._group_confidence_estimates = {}
         append_to_group = False
         group_file = "group_psms.csv"
         for group, group_df in data.groupby(self.group_column):
             LOGGER.info("Group: %s == %s", self.group_column, group)
             tdc_winners = group_df.index.intersection(idx)
             group_psms = group_df.loc[tdc_winners, :]
-            group_scores = scores.loc[group_psms.index].values + 1
+            group_scores = scores.loc[group_psms.index].values
```

Comment:
why is this correction factor not needed anymore?

Reply:
That was also not intented,it was from the last sync with original repo, the '+1' is incorrect